src/Metadata/AbstractPropertyMetadata.php

```diff
@@ -139,4 +180,9 @@ protected function setPublic(bool $public): void
     {
         $this->public = $public;
     }
+
+    protected function setCustomInformation(string $key, $value): void
+    {
+        $this->customInformation[$key] = $value;
```

Comment:
Should we prevent overwriting already existing custom information?

Reply:
i would not prevent that, it could be intended.